c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -2435,7 +2435,7 @@ class NativeCachingAllocator : public CUDAAllocator {
         fraction,
         ". Please set within (0, 1).");
     int activated_device;
-    C10_CUDA_CHECK(c10::cuda::GetDevice(&activated_device));
+    C10_CUDA_CHECK(cudaGetDevice(&activated_device));
```

Comment:
calling `cudaGetDevice` anywhere in torch seems wrong? Isn't the whole point that we are now supposed to use only `c10` functions, so that the tracked index is properly returned?